sci-biology/maq/maq-0.7.1-r4.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="Mapping and Assembly with Qualities, mapping NGS reads to reference genomes"
+HOMEPAGE="https://maq.sourceforge.net/"
+SRC_URI="
+	mirror://sourceforge/${PN}/${P}.tar.bz2
+	mirror://sourceforge/${PN}/calib-36.dat.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND="sys-libs/zlib"
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-flags.patch
+	"${FILESDIR}"/${P}-bfr-overfl.patch
+	"${FILESDIR}"/${P}-gcc-4.7.patch
+	"${FILESDIR}"/${P}-remove-64bit-flag.patch
+	"${FILESDIR}"/${P}-gcc14-build-fix.patch
```

Comment:
just bump the other ebuild, dont add a new one

Reply:
> All of these fixes require a new revision ("revbump") for the reasons described above. Also, developers want to know quickly if the fix is somehow insufficient, and a new revision helps to weed out any problems. 

From https://wiki.gentoo.org/wiki/Modern_C_porting#Fixes_.28C99.29

Also since I'm doing a runtime change, thought a revision bump would be needed.

Reply:
yes, but just add the patch and rename `-r3` -> `-r4`

Reply:
Ooh ... sure.

Reply:
Did the same for https://github.com/gentoo/gentoo/pull/36460